client/src/app/chat/_components/chat-box.tsx

```diff
@@ -0,0 +1,48 @@
+import { AvatarImage } from '@radix-ui/react-avatar'
+
+import { Avatar, AvatarFallback } from '@/components/avatar'
+import { Text } from '@/components/text'
+
+export interface ChatBoxProps {
+  photo: string
+  sellerName: string
+  date: Date
+  message: string
+  selected: boolean
+}
+
+function ChatBox({ photo, sellerName, date, message, selected }: ChatBoxProps) {
+  const dateTime = new Date(date)
+
+  return (
+    <div
+      className={
+        selected
+          ? 'flex h-20 w-full items-center gap-2 truncate bg-primary-50 p-2'
+          : 'flex h-20 w-full items-center gap-2 truncate p-2'
+      }
+    >
```

Comment:
มันซ้ำกันเยอะ อะไรที่ share กันก็รวบไว้ได้นะ

```suggestion
      className={cn("flex h-20 w-full items-center gap-2 truncate p-2", selected && 'bg-primary-50'}
    >
```